queries.sql

```diff
@@ -69,3 +69,20 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
 
 
+/* What animals belong to Melody Pond? */
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+/* List of all animals that are pokemon (their type is Pokemon). */
+SELECT animals.name FROM animals JOIN species ON animals.species_id  = species.id WHERE species.name = 'Pokemon';
+
```

Comment:
- [ ] **Required:** You are missing a required query for all names in the owners' table. Those that don't own any animals are inclusive.

Reply:
oh okay, LEFTJOIN. thank you . will make changes now